src/steamship/cli/create_instance.py

```diff
@@ -0,0 +1,119 @@
+import json
+from os import path
+from typing import Any, Dict, Optional
+
+import click
+
+from steamship import Steamship, SteamshipError
+from steamship.cli.ship_spinner import ship_spinner
+from steamship.data.manifest import DeployableType, Manifest
+
+
+@click.command(name="use")
+@click.option(
+    "--workspace",
+    "-w",
+    required=False,
+    type=str,
+    help="Workspace handle. The new instance will be created in this workspace. If not specified, "
+    "the default workspace will be used.",
+)
+@click.option(
+    "--instance",
+    "-i",
+    type=str,
+    required=False,
+    help="Instance handle. Used to name the instance (if it needs to be created). If not specified, "
+    "a generated instance name will be used.",
+)
+@click.option(
+    "--config",
+    "-c",
+    type=str,
+    required=False,
+    help="Instance configuration. May be inline JSON or a path to a file. If not specified, "
+    "an empty configuration dictionary will be passed to the instance.",
+)
+def create_instance(
+    workspace: Optional[str] = None, instance: Optional[str] = None, config: Optional[str] = None
+):
+    """Create an instance of your package/plugin for use.
+
+    Must be run from a directory containing a Steamship manifest."""
+    global client
+    global manifest
+
+    try:
+        client = Steamship(workspace=workspace)
```

Comment:
Instead of defaulting to the default workspace, should we default to one named after the package/plugin?  Multiple packages ending up in the same data space seems like a path to a lot of confusion

Reply:
This sounds like a reasonable plan to me (and I think matches the web-app UX?).  Will update.